webapp/locales/en.json

```
@@ -363,7 +363,11 @@
         "label": "Oldest first"
       }
     },
-    "order-by": "Order by ..."
+    "order-by": "Order by ...",
+    "save": {
+      "error": "Failed saving topic settings",
```

```suggestion
      "error": "Failed saving topic settings!",
```

We put an exclamation mark behind error, I guess.